go.mod

```diff
@@ -53,10 +53,14 @@ require (
 require (
 	github.com/aquasecurity/trivy v0.0.0-00010101000000-000000000000
 	github.com/containers/image/v5 v5.22.0
-	github.com/notaryproject/notation-go v0.10.0-alpha.3
+	github.com/notaryproject/notation v0.11.0-alpha.4
```

Comment:
You shouldn't need the `notation` since it's only for CLI exection. 
If you need `notation-core-go` - the notation team would like to know why as notation-go should be all you need to do whatever you need to do.

Reply:
@roxanaN FYI a little context wrt @dtzar 's suggestion.
 https://pkg.go.dev/github.com/notaryproject/notation-go@main